libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SyncTarget.java

```diff
@@ -26,110 +26,19 @@
  */
```

Comment:
Old SyncTarget is now SyncDownTarget
SyncTarget is now a simple interface with asJson as method.